pkg/openscap/openscap.go

```diff
@@ -142,7 +142,16 @@ func (s *defaultOSCAPScanner) getInputCVE(dist int) (string, error) {
 	}
 	defer out.Close()
 
-	resp, err := http.Get(cveURL.String())
+	client := &http.Client{}
+	req, err := http.NewRequest("GET", cveURL.String(), nil)
+	if err != nil {
+		return "", fmt.Errorf("Could not request file %s:\n", err)
+	}
+
+	// Some servers require 'Accept' and 'User-Agent' headers.
+	req.Header.Add("Accept", `text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8`)
```

Comment:
@nimrodshn I thought we don't accept `text/html`, is it strictly required?

If so, are you planning to catch that later on in the code and fail (e.g. if server returns `text/html` then print as debug and fail because we can't process it)